src/main/java/org/mskcc/cmo/messaging/impl/StreamingErrorListener.java

```diff
@@ -0,0 +1,23 @@
+package org.mskcc.cmo.messaging.impl;
+
+import io.nats.client.Connection;
+import io.nats.client.Consumer;
+import io.nats.client.ErrorListener;
+
+class StreamingErrorListener implements ErrorListener {
+    @Override
+    public void errorOccurred(Connection conn, String error) {
+        System.out.println("The server notificed the client with: " + error);
```

Comment:
can you add java docs here? looks like one of these is for server errors and the other. is connection errors?